### PR TITLE
[Documentation] Additional paramenters

### PR DESCRIPTION
**Pull request by peppelinux**
fixes https://github.com/IdentityPython/pysaml2/issues/710
minor change that hopefully make everyone happy ;)

### CODE REVIEW

**Comment by peppelinux** [comment]
In the second commit I added all the things I learned in the past two years of pysaml2 usage.

- Metadata MDQ/Remote `disable_ssl_certificate_validation`
- allow_unknown_attributes
- want_authn_requests_only_with_valid_cert
- only_use_keys_in_metadata
- validate_certificate
- encryption_keypairs
- a more detailed section, with examples, for Entity Categories

This commit also close this:
https://github.com/IdentityPython/pysaml2/issues/706

File: docs/howto/config.rst [hunk 1]
```diff
@@ -435,6 +467,11 @@ sign_response
 Specifies if the IdP should sign the authentication response or not. Can be
 True or False. Default is False.
 
+want_authn_requests_only_with_valid_cert
+""""""""""""""""""""""""""""""""""""""""
+
+When verifying a signed AuthnRequest ignore the signature and verify the
+certificate.
```
Comment (c00kiemon5ter) [inline docs/howto/config.rst:474]: I think this configuration option should be removed in the future.

File: docs/howto/config.rst [hunk 2]
```diff
@@ -881,6 +917,19 @@ Example::
         },
     },
 
+only_use_keys_in_metadata
+"""""""""""""""""""""""""
+
+If set to False, the certificate contained in a SAML message will be used for
+signature verification.
+Default True.
```
Comment (c00kiemon5ter) [inline docs/howto/config.rst:925]: I think this configuration option should be removed in the future. We should not be using anything but the certificates we know about.

File: docs/howto/config.rst [hunk 3]
```diff
@@ -881,6 +917,19 @@ Example::
...
+
+validate_certificate
+""""""""""""""""""""
+
+Indicates that the certificate used in sign SAML messages must be valid.
+Default to False.
```
Comment (c00kiemon5ter) [inline docs/howto/config.rst:931]: I think this configuration option should change to be True by default.

**Pull request merged**